### ISSUE TITLE
Create Marker modal needs overflow: hidden

### ISSUE THREAD

**Issue by kevinwhoffman** [WordImpress/Google-Maps-Builder#202]
The Create Marker modal window background does not fill the entire container. Here is the CSS to fix it:
```
.magnific-builder #poststuff .white-popup .inner-modal-wrap .inner-modal {
    overflow: hidden;
}
```

**Before overflow: hidden**
![create-marker-before-overflow](https://cloud.githubusercontent.com/assets/1938671/16674688/d68c4444-4488-11e6-8288-adbae51dc280.jpg)

**After overflow: hidden**
![create-marker-after-overflow](https://cloud.githubusercontent.com/assets/1938671/16674691/dbfd9c5c-4488-11e6-9d47-9ca3c0801daf.jpg)

**Comment by DevinWalker** [comment]
Overflow hidden doesn't make the location dropdown disappear? How about adding a clearfix class to the modal container?

**Comment by kevinwhoffman** [comment]
There is already a clearfix class on the container but it doesn't seem to be doing anything. I don't remember seeing this visual bug in previous versions, so maybe something changed in 2.0.2.

Here's the change I made that fixed it for me. Note the clearfix class is already on the element where I've added `overflow: hidden`.
![create-marker-dev-tools](https://cloud.githubusercontent.com/assets/1938671/16698675/7730fa72-451d-11e6-9473-c3d633f53e36.jpg)
